scylla-cdc-replicator/src/replicator_consumer.rs

```diff
@@ -1,3 +1,5 @@
+use std::sync::Arc;
+
```

Comment:
One more thing: it's better to avoid unnecessary changes like moving around imports too much or adding/removing unrelated whitespace. It makes it harder to review such commits and can make it harder for others to rebase on top of your changes. If you still think there is a good reason to make the whitespace change, it's better to separate it from functional changes and put them in a separate commit.

Reply:
I think some of the whitespace changes are the result of `cargo fmt`.